.github/workflows/docker.yml

```diff
@@ -64,8 +57,7 @@ jobs:
             name=ghcr.io/${{ github.repository_owner }}/${{ matrix.image }}-${{ matrix.target }}
           tags: |
             # Tag scheduled runs with date
-            type=schedule,pattern={{date 'YYYYMMDD'}}
-            type=raw,value={{date 'YYYYMMDD'}},enable=${{ inputs.pushDateTag || false }}
+            type=raw,value=${{ github.ref_name }},enable=${{ github.event_name == 'push' }}
             # Tag with branch name
             type=ref,event=branch
             # Tag with pr-number
```

Comment:
I think we can just remove the whole tags block now and use the default (https://github.com/marketplace/actions/docker-metadata-action#tags-input)

Reply:
We still want to use the git tag value

Reply:
The defaults include ``type=ref,event=tag`, which should use the git tag value when relevant.